crosscomponents.cmake

```diff
@@ -7,4 +7,13 @@ set (CLR_CROSS_COMPONENTS_LIST
   sos
   clrjit
   protojit
-)  
+)
+
+if(CLR_CMAKE_PLATFORM_LINUX AND CLR_CMAKE_TARGET_ARCH_ARM)
```

Comment:
@janvorli OK. I'll modify to set list for ARM using if-else. 

By building these projects, we can get cross-architecture crossgen and superpmi (release build). And to building other projects, need to fix more build error.